### PR TITLE
OCC version<7.3 is never used in freecad0.20

### PR DESCRIPTION
**Pull request by mosfet80**
Thank you for creating a pull request to contribute to FreeCAD! To ease integration, we ask you to conform to the following items. Pull requests which don't satisfy all the items below might be rejected. If you are in doubt with any of the items below, don't hesitate to ask for help in the [FreeCAD forum](https://forum.freecadweb.org/viewforum.php?f=10)!

- [ ]  Your pull request is confined strictly to a single module. That is, all the files changed by your pull request are either in `App`, `Base`, `Gui` or one of the `Mod` subfolders. If you need to make changes in several locations, make several pull requests and wait for the first one to be merged before submitting the next ones
- [ ]  In case your pull request does more than just fixing small bugs, make sure you discussed your ideas with other developers on the FreeCAD forum
- [x]  Your branch is [rebased](https://git-scm.com/docs/git-rebase) on latest master `git pull --rebase upstream master`
- [ ]  All FreeCAD unit tests are confirmed to pass by running `./bin/FreeCAD --run-test 0`
- [ ]  All commit messages are [well-written](https://chris.beams.io/posts/git-commit/) ex: `Fixes typo in Draft Move command text`
- [ ]  Your pull request is well written and has a good description, and its title starts with the module name, ex: `Draft: Fixed typos`
- [ ]  Commit messages include `issue #<id>` or `fixes #<id>` where `<id>` is the issue ID number from our [Issues database](https://github.com/FreeCAD/FreeCAD/issues) in case a particular commit solves or is related to an existing issue. Ex: `Draft: fix typos - fixes #4805`

And please remember to update the Wiki with the features added or changed once this PR is merged.  
**Note**: If you don't have wiki access, then please mention your contribution on the [0.20 Changelog Forum Thread](https://forum.freecadweb.org/viewtopic.php?f=10&t=56135).

---

### CODE REVIEW

**Comment by freecadci** [comment]
<a href="https://gitlab.com/freecad/FreeCAD-CI/-/commits/PR_7037">pipeline status</a> for feature branch [PR_7037](https://gitlab.com/freecad/FreeCAD-CI/-/commits/PR_7037). Pipeline [566224034](https://gitlab.com/freecad/FreeCAD-CI/-/pipelines/566224034) was triggered at [8d9f67e](https://github.com/FreeCAD/FreeCAD/pull/7037/commits/8d9f67eff1bcea8285ab537ad9652480faa9fc04). All CI [branches](https://gitlab.com/freecad/FreeCAD-CI/-/branches/all) and [pipelines](https://gitlab.com/freecad/FreeCAD-CI/-/pipelines?scope=branches).

**Comment by freecadci** [comment]
<a href="https://gitlab.com/freecad/FreeCAD-CI/-/commits/PR_7037">pipeline status</a> for feature branch [PR_7037](https://gitlab.com/freecad/FreeCAD-CI/-/commits/PR_7037). Pipeline [567093122](https://gitlab.com/freecad/FreeCAD-CI/-/pipelines/567093122) was triggered at [6d86a1c](https://github.com/FreeCAD/FreeCAD/pull/7037/commits/6d86a1c01490434a7adb5c8ea2b313e4b085faeb). All CI [branches](https://gitlab.com/freecad/FreeCAD-CI/-/branches/all) and [pipelines](https://gitlab.com/freecad/FreeCAD-CI/-/pipelines?scope=branches).

File: src/Mod/Drawing/App/DrawingExport.cpp [hunk 1]
```diff
@@ -73,7 +72,9 @@
 #include <BRepLProp_CLProps.hxx>
 #include <Standard_Failure.hxx>
 #include <Standard_Version.hxx>
-
+#if OCC_VERSION_HEX < 0x070600
```
Comment (donovaly) [inline src/Mod/Drawing/App/DrawingExport.cpp:75]: Why this code addition? I think the aim of the PR is to remove code from OCC < 7.3

Please make a separate PR for the additions because just in case something unforeseen happens we can distinguish the commits.

**Review by donovaly — CHANGES_REQUESTED** [review]
Please make a separate PR for the additions because just in case something unforeseen happens we can distinguish the commits.

**Comment by yorikvanhavre** [comment]
Same as the other PR... i don't really see a good reason to do these limitations, unless of course it makes our work easier

**Comment by donovaly** [comment]
At least the OCC 6 code should go. I think it is not sensible to have code for ancient versions that nobody uses. For OCC 7.x I agree to keep it for now.

**Comment by freecadci** [comment]
<a href="https://gitlab.com/freecad/FreeCAD-CI/-/commits/PR_7037">pipeline status</a> for feature branch [PR_7037](https://gitlab.com/freecad/FreeCAD-CI/-/commits/PR_7037). Pipeline [567924264](https://gitlab.com/freecad/FreeCAD-CI/-/pipelines/567924264) was triggered at [9b769d9](https://github.com/FreeCAD/FreeCAD/pull/7037/commits/9b769d9b9032c069f40ee3b868dfb8c994d9e5c5). All CI [branches](https://gitlab.com/freecad/FreeCAD-CI/-/branches/all) and [pipelines](https://gitlab.com/freecad/FreeCAD-CI/-/pipelines?scope=branches).

**Comment by freecadci** [comment]
<a href="https://gitlab.com/freecad/FreeCAD-CI/-/commits/PR_7037">pipeline status</a> for feature branch [PR_7037](https://gitlab.com/freecad/FreeCAD-CI/-/commits/PR_7037). Pipeline [567990339](https://gitlab.com/freecad/FreeCAD-CI/-/pipelines/567990339) was triggered at [5771aa0](https://github.com/FreeCAD/FreeCAD/pull/7037/commits/5771aa06a0b1a3339796fc5f189ea90fbd8de123). All CI [branches](https://gitlab.com/freecad/FreeCAD-CI/-/branches/all) and [pipelines](https://gitlab.com/freecad/FreeCAD-CI/-/pipelines?scope=branches).

**Comment by freecadci** [comment]
<a href="https://gitlab.com/freecad/FreeCAD-CI/-/commits/PR_7037">pipeline status</a> for feature branch [PR_7037](https://gitlab.com/freecad/FreeCAD-CI/-/commits/PR_7037). Pipeline [568154644](https://gitlab.com/freecad/FreeCAD-CI/-/pipelines/568154644) was triggered at [f0b2121](https://github.com/FreeCAD/FreeCAD/pull/7037/commits/f0b21214695e7d9fe8bfdcbb3d69ecb14a767206). All CI [branches](https://gitlab.com/freecad/FreeCAD-CI/-/branches/all) and [pipelines](https://gitlab.com/freecad/FreeCAD-CI/-/pipelines?scope=branches).

**Comment by donovaly** [comment]
@mosfet80 , as written, please make a new PR that only removes OCC < 7. Let's keep support for OFF 7.x for now.

There is also something from with the PR since it has now 26 commits.
